library/CoreData/Base/iCloudCDStack.swift

```diff
@@ -333,7 +335,7 @@ public class iCloudCDStack: DefaultCDStack
     
     :param: notification Notification with these changes
     */
-    internal func persistentStoreDidImportUbiquitousContentChanges(notification: NSNotification)
+    @objc internal func persistentStoreDidImportUbiquitousContentChanges(notification: NSNotification)
```

Comment:
Yes, this is what was causing issue: https://github.com/SugarRecord/SugarRecord/issues/129